subsys/bluetooth/controller/ll_sw/nordic/lll/lll_conn.c

```diff
@@ -491,11 +554,16 @@ void lll_conn_isr_tx(void *param)
 
 	radio_tmr_hcto_configure(hcto);
 
-#if defined(CONFIG_BT_CENTRAL) && defined(CONFIG_BT_CTLR_CONN_RSSI)
+#if defined(CONFIG_BT_CTRL_DF_CONN_CTE_RX)
+	if (!trx_cnt) {
+#elif defined(CONFIG_BT_CENTRAL) && defined(CONFIG_BT_CTLR_CONN_RSSI)
 	if (!trx_cnt && !lll->role) {
+#else
+	if (0) {
+#endif /* CONFIG_BT_CTRL_DF_CONN_CTE_RX */
+
```

Comment:
Dont we want rssi for all Rx-ed PDU?

```suggestion
#if defined(CONFIG_BT_CTRL_DF_CONN_CTE_RX)
	if (true) {
#elif defined(CONFIG_BT_CENTRAL) && defined(CONFIG_BT_CTLR_CONN_RSSI)
	if (!trx_cnt && !lll->role) {
#else
	if (0) {
#endif /* CONFIG_BT_CTRL_DF_CONN_CTE_RX */

```

Reply:
Isn't it also a bit odd to use `0` for false, and `true` for true, instead of using true/false, or 1/0?

Reply:
Corrected with use of true/false in both if statements.